app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadViewModel.kt

```diff
@@ -70,6 +71,29 @@ class ThreadViewModel @Inject constructor(
         messageController.getSortedMessages(threadUid)?.asFlow()?.asLiveData()?.let { emitSource(it) }
     }
 
+    private val currentMailboxLive = MailboxController.getMailboxAsync(
+        AccountUtils.currentUserId,
+        AccountUtils.currentMailboxId,
+    ).map { it.obj }.asLiveData(ioCoroutineContext)
+
+    fun threadMergedContactAndMailboxMediator(
```

Comment:
The name is kinda cluttered.

Maybe `fun threadAndMergedContactAndMailboxMediator()` ?